sdkproject/Assets/Mapbox/Unity/MeshGeneration/LayerVisualizers/LocationPrefabsLayerVisualizer.cs

```diff
@@ -0,0 +1,309 @@
+namespace Mapbox.Unity.MeshGeneration.Interfaces
+{
+	using System;
+	using System.Collections.Generic;
+	using Mapbox.Unity.Utilities;
+	using Mapbox.Unity.MeshGeneration.Filters;
+	using UnityEngine;
+	using Mapbox.Unity.MeshGeneration.Modifiers;
+	using Mapbox.Unity.MeshGeneration.Data;
+	using Mapbox.Unity.Map;
+	using Mapbox.VectorTile;
+
+	public class LocationPrefabsLayerVisualizer : VectorLayerVisualizer
+	{
+		private int maxDensity = 30; //This value is same as the density's max range value in PrefabItemOptions
+
+		public void SetProperties(PrefabItemOptions item, LayerPerformanceOptions performanceOptions)
+		{
+			SubLayerProperties = item;
+			_performanceOptions = performanceOptions;
+
+			if (!item.isActive)
+				return;
```

Comment:
Shouldn't the properties be set regardless of the "Active" status of the layer ?